package.json

```diff
@@ -3,38 +3,25 @@
   "version": "1.0.0",
   "description": "Leather wallet shared code",
   "scripts": {
-    "install:clean": "rm -rf \"**/node_modules\" && pnpm -r clean",
-    "install:fresh": "pnpm install:clean && pnpm i",
-    "install:nuke": "rm -rf pnpm-lock.yaml && pnpm install:fresh",
-    "lint:eslint": "eslint .",
-    "lint:eslint:fix": "pnpm lint:eslint --fix",
-    "lint:prettier": "prettier --check \"*.{ts,tsx,js,json}\"",
-    "lint:prettier:fix": "prettier --write \"*.{ts,tsx,js,json}\" *.js",
-    "lint:types": "tsc --noEmit",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "lint-projects:all": "nx run-many -t format,lint,typecheck --parallel=1",
+    "lint-projects:affected": "npx nx affected -t format,lint,typecheck --parallel=1 --base=dev",
+    "commitlint": "commitlint"
   },
   "keywords": [],
   "author": "",
   "license": "MIT",
+  "workspaces": [
+    "packages/*"
```

Comment:
I missed this earlier but I think that the mobile app should live under `apps`

So we will have:
- `apps/*`
- `packages/*`

There is different conventions calling things `packages` or `libs`. I was considering renaming them as `libs` as then this [workflow tool ](https://kevinkreuzer.medium.com/how-to-release-npm-libraries-with-nx-461f7671a96f)could work for us

Reply:
Great point! Moved to apps/ folder